Doc/library/http.client.rst

```diff
@@ -67,10 +67,9 @@ The module provides the following classes:
       *blocksize* parameter was added.
 
 
-.. class:: HTTPSConnection(host, port=None, key_file=None, \
-                           cert_file=None[, timeout], \
+.. class:: HTTPSConnection(host, port=None[, timeout], \
```

Comment:
Can't leave a suggestion, but this one needs to be made keyword-only

Reply:
Updated in 25421ff599539ccf702c56a13475eb381394b63e